src/universal/HTTPS-README.md

```diff
@@ -0,0 +1,26 @@
+For using JSON-RPC through HTTPS a certificate has to be created. For this first a password has to be created:
```

Comment:
Kudos for the explanation :heart: . That being said, I would put this within the wiki and a link to it in `src/universal/conf/network.conf`